src/InteractiveIframe/iframeCard.jsx

```diff
@@ -23,8 +23,9 @@ export default class IframeCard extends Component {
   }
 
 	getSrcURL = () => {
-	  if (this.props.params) {
-	    return UrlUtils.template(this.props.url, this.props.params);
+	  if (this.props.params || this.props.externalProps) {
+      const params = this.props.params || this.props.externalProps;
```

Comment:
Looks like we're preferring local params rather than those that come from external.  Is that the correct preference?  I would've thought we'd prefer external.

Reply:
looks like the code is not allgined in this line.

Reply:
Why params are picked and not both?

Reply:
@ctpaterson ... params is not a 'local' param, it is an external property that is passed into the component. Mosaic and the DOP solution both evolved on top of this configurable-interactive-layout component which resulted in misaligned parameter names (Mosaic added support for externalProps without integrating into the iFrameCard ... now coming back, we have two properties being passed into the iFrameCard that do the same thing). Bottom line, props.params was the original design, props.externalProps came second from Mosaic. We could eventually have a feature to consolidate these values, but I suspect applications will choose to use only one of these properties, not both (we can schedule a patch later on if there is value in consolidating, but there will be in depth discussions needed to figure out which takes precedence when there is duplicate keys in both (especially if there is a nested structure)

@sali1982 ... indenting here seems to not match what's in my IDE ... no idea why const is not indented here.

Reply:
Then just Capture this discussion in Jira please.